definitions/variable/emissions/tag_long-lived-materials.yaml

```diff
@@ -0,0 +1,7 @@
+- Long-Lived Materials:
+    - Timber:
+        description: long-lived wood products
```

Comment:
It would be useful to provide indicative time frames (e.g., "timber in construction and other wood products which lasts more than 100 years without decomposing")

Reply:
Added the same sentence as in https://github.com/IAMconsortium/common-definitions/blob/be138400d429363066c497535a1ab5af734f9042/definitions/variable/emissions/tag_carbon-utilization.yaml#L5